README.md

```diff
@@ -144,37 +167,46 @@ client.setRefreshToken(AMAZON_DRS_REFRESH_TOKEN);
 
 ### getRefreshToken() ###
 
-The method returns a string with the `Refresh Token` or `null` if it is not set.
+The method returns a string with the Refresh Token, or `null` if it has not been set.
+
+#### Return Value ####
+
+String &mdash; The Refresh Token, or `null`.
 
 ### replenish(*slotId[, onReplenished]*) ###
 
 This method places an order for a device/slot combination. For more information, please see [the Amazon DRS documentation](https://developer.amazon.com/docs/dash/replenish-endpoint.html).
 
-| Parameter | Data Type | Required? | Description |
-| --- | --- | --- | --- |
-| *slotId* | String | Yes | ID of a slot to place an order for it. |
-| *onReplenished* | Function | Optional | Callback called when the operation is completed or an error occurs. |
+#### Parameters ####
 
-The method returns nothing. A result of the operation may be obtained via the [onReplenished](#callback-onreplenishederror-response) callback if specified in this method.
+| Parameter | Type | Required | Description |
+| --- | --- | --- | --- |
+| *slotId* | String | Yes | The ID of a slot used to place an order |
+| *onReplenished* | Function | Optional | Callback function called when the operation is completed or an error occurs |
 
-#### Callback: onReplenished(*error, response*) ####
+#### onReplenished Callback Parameters ####
 
 | Parameter | Data Type | Description |
 | --- | --- | --- |
-| *error* | Integer | `0` if the operation is completed successfully, an [error code](#error-code) otherwise. See possible HTTP error codes [here](https://developer.amazon.com/docs/dash/replenish-endpoint.html#error-responses). |
-| *response* | Table | Key-value table with the response provided by Amazon server. May be `null`. [See response example](https://developer.amazon.com/docs/dash/replenish-endpoint.html#response-example). Also may contain [error details](https://developer.amazon.com/docs/dash/replenish-endpoint.html#error-responses). |
+| *error* | Integer | `0` if the authentication is successful, otherwise an [error code](#error-code) otherwise. Possible HTTP error codes [are listed here](https://developer.amazon.com/docs/dash/replenish-endpoint.html#error-responses) |
```

Comment:
`authentication` - I suppose it is a copy-paste mistake. And `otherwise` written twice as above.

Reply:
Fixed